CONTRIBUTING.md

```diff
@@ -249,7 +252,7 @@ Example:
     // See the License for the specific language governing permissions and
     // limitations under the License.
 ```
-
+**_NOTE:_**  For scripts or CMake files you can use `#` instead of `//` for the header.
```

Comment:
how about inserting a "... you can use the respective comment syntax like `#` ..."